backend/main.go

```diff
@@ -46,5 +55,50 @@ func HealthProbe(response http.ResponseWriter, request *http.Request) {
 func GetEvents(response http.ResponseWriter, request *http.Request) {
 	payload := "Fetch stuff from the database after we get models."
 	logrus.Infoln("handled request on /events ")
+
 	respondWithJSON(response, http.StatusOK, payload)
 }
+
+func GetUsers(response http.ResponseWriter, request *http.Request) {
+	payload := "Fetch users from the database after we get models."
+	logrus.Infoln("handled request on /users ")
+	user := models.UserModel{}
+	fmt.Println(user)
+
+	respondWithJSON(response, http.StatusOK, payload)
+}
+
+func PostUser(w http.ResponseWriter, r *http.Request) {
+	var userJson models.UserJSON
+	userID := uuid.New()
```

Comment:
Can we use Postgresqls internal UUID generation here to remove extra logic from our service?

Reply:
On doing some more research this varies per Postgresql version so maybe this is the "most stable/compatible" way to do it.